addon/services/construction-project.js

```diff
@@ -139,4 +752,8 @@ export default class ConstructionProjectService extends GwrService {
       }
     });
   }
+
+  getChangeHint(currentStatus, newStatus) {
+    return ConstructionProject.projectTransitionHint[currentStatus][newStatus];
+  }
```

Comment:
Maybe put this in gwr base service.
```suggestion
  getChangeHint(currentStatus, newStatus) {
    return this.cacheClass.projectTransitionHint[currentStatus][newStatus];
  }
```